app/src/ui/create-branch/create-branch-dialog.tsx

```diff
@@ -230,11 +230,10 @@ export class CreateBranch extends React.Component<
   }
 
   public render() {
-    const proposedName = this.state.proposedName
     const disabled =
-      !proposedName.length ||
+      !this.state.proposedName.length ||
```

Comment:
Could you explicitly check length here instead of relying on JS's truthiness?

`this.state.proposedName.length <= 0`

Reply:
Fixed in 1c9a19cc79d3030629248def68bdb96ec4bc61e4. :mag: